### DISCUSSION TITLE
Custom Calendar, Chart and Graph widgets

### DISCUSSION THREAD

**Discussion by ZikPin** [TomSchimansky/CustomTkinter#1836] [Showcase]
After showcasing my previous project ([Productivity App](https://github.com/ZikPin/Productivity-App)), Akascape had an interesting idea to release some widgets independently, not as part of the project, so that people could use them. I tried and here are some that I wrote:
_Default view is on the left, and customized version on the right (might not be pretty but you get the point :>) and the repository [-CTkDataVisualizingWidgets](https://github.com/ZikPin/-CTkDataVisualizingWidgets)_
- The calendar frame lets you make customizable calendar frame with buttons to change month

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/69543cae-f78e-4ad8-846c-afcdb7935fec)

- The graph widget helps you visualize some data (lists of integers) and is also customizable

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/e51afa6a-7370-411f-b491-c25f4dffcd33)

- The chart works kind of like graph but works with dictionaries and displays both value and the keys.

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/73c78ca6-a0ec-4674-ab66-eeb497140ae4)

For more info you can check out my repository [-CTkDataVisualizingWidgets](https://github.com/ZikPin/-CTkDataVisualizingWidgets). Probably in the future will try to add more widgets. If you have some issues or bugs pls notify me.

**Comment by Akascape** [comment]
@ZikPin Thanks for releasing them independently. Really a great library for data visualisation. Maybe change the name of the repo to something more unique, as the widgets are related to data visualisation, why not call them something like CTkDataViewWidgets or simply CTkDataWidgets.?

**Reply by avalon60** [reply]
Cool widgets! 👍

I seem to recall that someone was asking about a calendar widget some time back. Nice addition.
